WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditOrderAddressForm.swift

```diff
@@ -270,6 +273,19 @@ struct SingleAddressForm: View {
 
             }
 
+            if showPhoneCountryCodeField {
+                TitleAndTextFieldRow(title: Localization.phoneCountryCodeField,
+                                     titleWidth: $titleWidth,
+                                     placeholder: Localization.phoneCountryCodeHint,
+                                     text: $fields.phoneCountryCode,
+                                     symbol: nil,
+                                     fieldAlignment: .leading,
+                                     keyboardType: .phonePad)
```

Comment:
Nit: we can use `asciiCapableNumberPad` since we expect only digits in this field.

Reply:
TIL thanks for the suggestion, this is the perfect keyboard type for this use case. Updated in https://github.com/woocommerce/woocommerce-ios/pull/8859/commits/2d0b25a17ac6cd5c6a52c3ceab78551aa69e8309